main.js

```diff
@@ -55,4 +55,4 @@ function updateTwitterIntent(quote) {
   var intentAddress = 'https://twitter.com/intent/tweet?text=';
   intentAddress += '\"' + quote['text'] + '\"' + '  — ' + quote['movie'];
   document.getElementById('tweet-quote-link').setAttribute('href', intentAddress);
-}
\ No newline at end of file
+}
```

Comment:
nice work on the title of the commits. it is really nice that you take the time to write nice captions.
don't be afraid to commit more often to avoid unrelated content (e.g. `Update button styles` also includes changes to the quote data. I often do multiple commits after a programming stretch.